### DISCUSSION TITLE
Ham Radio Deluxe Logbook interface available ?

### DISCUSSION THREAD

**Discussion by myhobby2** [magicbug/Cloudlog#3140] [Third Party Tools]
Just wondering if there is a setup available that copies the HRD-logbook into my central log app (ofcourse this is Cloudlog). Running cloudlog in my local network as a virtual machine (and own IP address). Is there a setup instruction available how to configure HRD-logbook so it writes in the cloudlog sql database ?

**Comment by magicbug** [comment]
Just had a look at the HRD manual and it seems to have QSO Forwarding see https://support.hamradiodeluxe.com/support/solutions/articles/51000052684-qso-forwarding if you run Cloudlog Aurora and enable WSJT-X functionality I've a feeling with HRD configured to **UDP Send** it might be possible to forward QSOs to Cloudlog

**Reply by magicbug** [reply]
Worth noting Cloudlog Aurora uses UDP port 2237

**Reply by myhobby2** [reply]
Thanks for the help provided. I'm new to HRD and still looking for this setting. Good to know the port number, and the suggesting the WSJT-X functionality. For the moment I can't run HRD and WSJT-X parallel at the same time because both will require exclusive control over the CAT Hardware. This excludes WSJT-X as the intermediar between HRD and Cloudlog...

**Reply by MikleBas** [reply]
For these purposes (COM port sharing), special software has long been written, this program works best for me
Virtual Serial Ports Emulator (VSPE)
(paid, but it's worth it) - but there are also free options, although none of them worked for me :)
73!
